src/theme/dark.ts

```diff
@@ -260,9 +260,10 @@ const popOver = {
   titleText: colors.greys.white,
   active: {
     itemBgd: colors.primary.dark,
-    itemText: colors.greys.grey80
+    itemText: colors.greys.grey80,
   },
-  separator: colors.greys.grey100
+  disabled: colors.greys.grey40,
```

Comment:
The disabled grey color in dark and light theme is same, checking if checked or should we check with UX once on this?

Reply:
The color of this disabled props is the same setting as `button`. And the `PopOverItem` uses `button` style. So it should be fine.